src/node_api.h

```diff
@@ -614,6 +614,36 @@ NAPI_EXTERN napi_status napi_run_script(napi_env env,
 NAPI_EXTERN napi_status napi_get_uv_event_loop(napi_env env,
                                                struct uv_loop_s** loop);
 
+// Calling into JS from other threads
+NAPI_EXTERN napi_status
+napi_create_threadsafe_function(napi_env env,
+                                napi_value func,
+                                napi_value async_resource,
+                                napi_value async_resource_name,
+                                size_t max_queue_size,
+                                size_t initial_refcount,
+                                void *thread_finalize_data,
+                                napi_finalize thread_finalize_cb,
+                                void *hint,
```

Comment:
This should be `void* context` to match the impl and the method that returns the context.